### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by gdozot2**

### CODE REVIEW

File: src/api/dataFetching.js [hunk 1]
```diff
@@ -5,76 +5,72 @@ const request = async (url) => {
...
 
 export default {
-  async getImages(listId) {
-    let req = process.env.VUE_APP_FETCH_BASE;
-    let position = 3;
-    listId.forEach((element) => {
+  async getImages(user, skipId) {
+    let req =
+      process.env.VUE_APP_FETCH_BASE +
+      "per_page=1&property%5B0%5D%5Bjoiner%5D=and&property%5B0%5D%5Bproperty%5D=15&property%5B0%5D%5Btype%5D=nex&property%5B1%5D%5Bjoiner%5D=and&property%5B1%5D%5Bproperty%5D=6&property%5B1%5D%5Btype%5D=eq&property%5B1%5D%5Btext%5D=" +
```
Comment (slecorne) [inline src/api/dataFetching.js]: would be much better to use axios here with object `params` to follow the parameter you sent. It is hard to know here with parameters hidden in the string + html encoding:
Same for others GET

File: src/App.vue [hunk 2]
```diff
@@ -329,25 +483,136 @@ export default {
...
+      this.isLoad = false;
+    },
+    loadImagePerDay() {
+      dataFetch.getTodayImagessByUser(this.user).then((result) => {
+        this.todayImages = result.nbImage;
+        this.todayTags = result.nbTags;
+      });
+    },
+    loadData() {
+      let skipId = undefined;
```
Comment (slecorne) [inline src/App.vue:592]: needs comment, really hard to follow

File: src/App.vue [hunk 3]
```diff
@@ -329,25 +483,136 @@ export default {
...
     },
     removeEntry(position) {
       this.selectedTags.splice(position, 1);
     },
     skipTags() {
-      this.skipId.push(
-        this.images[0].element["dcterms:identifier"][0]["@value"]
-      );
-      this.skipId = [...new Set(this.skipId)];
       this.images[0].element["dcterms:rights"] = [
```
Comment (slecorne) [inline src/App.vue:641]: why `images` and then using `images[0]`?
also skipId seems only one id, how does it work if multiple skip images?

File: src/api/dataFetching.js [hunk 4]
```diff
@@ -83,7 +79,81 @@ export default {
...
+    const today = new Date();
+    const date =
+      today.getDate() +
+      "-" +
+      (today.getMonth() + 1) +
+      "-" +
+      today.getFullYear();
+    let req;
+    const dataProcessing = [];
+    for (let i = 1; i <= 3; i++) {
```
Comment (slecorne) [inline src/api/dataFetching.js:101]: please comment why 3

File: src/App.vue (hunk 3 above)
Comment (gdozot2) [reply src/App.vue:641]: The skipId is used to skip the current image and preload the next image

**Review by slecorne — APPROVED** [review]
